src/panels/config/person/dialog-person-detail.ts

```diff
@@ -191,8 +191,10 @@ class DialogPersonDetail extends LitElement {
       const values: PersonMutableParams = {
         name: this._name.trim(),
         device_trackers: this._deviceTrackers,
-        user_id: this._userId || null,
       };
+      if (this._userId !== this._params!.entry!.user_id) {
```

Comment:
An alternative to this can probably be this?
```js
      if (this._params && this._params.entry && this._params.entry.user_id && this._userId !== this._userId) {
```

Reply:
That can be simplified to `this._params?.entry?.user_id`

Reply:
That looks way better, applied.